src/main/public/js/calculate-amount-total.js

```diff
@@ -12,15 +13,20 @@ $(document).ready(function () {
     return true
   }
 
-  $('input.claim-amount-column-2').keyup(updateTotal)
+  $('#claim-amount').on('keyup','input.claim-amount-column-2', updateTotal)
 
   function toNumber (index, inputElement) {
-    var parsed = parseFloat($(inputElement).val())
+    var regex =  /^(\d+|\d{1,3}(,\d{3})*)(\.\d+)?$/;
```

Comment:
We validate that the input has a maximum of 2dp. We could put that in this regex too
`... (\.\d\d?)?` or `... (\.\d{1,2})?`